src/client/Client.cc

```diff
@@ -1461,7 +1461,7 @@ mds_rank_t Client::choose_target_mds(MetaRequest *req, Inode** phash_diri)
       }
     }
   
-    if (in->auth_cap && req->auth_is_best()) {
+    if (in->auth_cap && (req->auth_is_best() || req->send_to_auth)) {
```

Comment:
how about add it to MetaRequest::auth_is_best()

Reply:
Of course:), will update it right now.